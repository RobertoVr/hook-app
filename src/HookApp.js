import React from 'react'

export const HookApp = () => {
    return (
        <div>
            Hooks works
        </div>
    )
}