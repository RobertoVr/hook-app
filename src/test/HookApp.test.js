import React from 'react';
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import { HookApp } from '../HookApp';

describe('Probando hook app', () => {
    test('Debe hacer render el hook app', () => {
        const wrapper = shallow(<HookApp />);
        expect(wrapper).toMatchSnapshot();
    })

})
