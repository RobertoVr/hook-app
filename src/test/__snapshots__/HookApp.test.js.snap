// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Probando hook app Debe hacer render el hook app 1`] = `
<div>
  Hooks works
</div>
`;
